spec/polyn_spec.rb

```diff
@@ -75,33 +74,34 @@
     end
 
     it "always includes a Nats-Msg-Id header" do
-      Polyn.publish(nats, "calc.mult.v1", {
+      subject.publish("calc.mult.v1", {
         a: 1,
         b: 2,
-      }, store_name: store_name, reply_to: "foo")
+      }, reply_to: "foo")
 
       msg = get_message("calc.mult.v1", "my_consumer", "CALC")
 
       expect(msg.header).to eq({ "Nats-Msg-Id" => JSON.parse(msg.data)["id"] })
     end
 
     it "can include a header" do
-      Polyn.publish(nats, "calc.mult.v1", {
+      subject.publish("calc.mult.v1", {
         a: 1,
         b: 2,
-      }, store_name: store_name, header: { "a header key" => "a header value" }, reply_to: "foo")
+      }, header: { "a header key" => "a header value" }, reply_to: "foo")
 
       msg = get_message("calc.mult.v1", "my_consumer", "CALC")
 
-      expect(msg.header).to eq({ "Nats-Msg-Id" => JSON.parse(msg.data)["id"], "a header key" => "a header value" })
+      expect(msg.header).to eq({ "Nats-Msg-Id" => JSON.parse(msg.data)["id"],
+"a header key" => "a header value" })
```

Comment:
Moar nitpick, but this should probably float out a little farther.

Reply:
:shakes fist at rubocop:

Reply:
https://github.com/SpiffInc/polyn-ruby/pull/13/commits/9f8f8623931543e0739796a55be9e41c23b46fb1